.env-devel

```diff
@@ -32,7 +32,7 @@ AUTOSCALING_NODES_MONITORING=null
 AUTOSCALING_POLL_INTERVAL=10
 AUTOSCALING_SSM_ACCESS=null
 
-AWS_S3_CLI_S3='{"S3_ACCESS_KEY":"12345678", "S3_BUCKET_NAME":"simcore", "S3_ENDPOINT": "http://172.17.0.1:9001", "S3_SECRET_KEY": "12345678", "S3_REGION": "us-east-1"}'
+AWS_S3_CLI_S3=null
```

Comment:
ah but this only disables locally, not in master.

Reply:
in master it was already changed in the ops deployments repo :+1: